### ISSUE TITLE
How to differentiate between registry when scopes are same ?

### ISSUE THREAD

**Issue by simbathesailor** [npm/cli#2307]
### Current Behavior:
I have two packages
 @simbathesailor/randomnumberlib from  registry : https://npm.pkg.github.com (private)
@simbathesailor/use-what-changed from registry : https://registry.npmjs.org/

Both the packages need to be consumed in the same repository.

How can I mention in the npmrc , that @simbathesailor/randomnumberlib  need to be fetched from github packages and 
@simbathesailor/use-what-changed from npm packages. 

Note: If the scope would have been different , I know how How i can mention it in .npmrc.

This might be a small query for some of them , but I tried looking for the soluition on npm docs and stackoverflow. But no help.

```
@simbathesailor:registry=https://npm.pkg.github.com/
@simbathesailorsomethingdifferent:registry=https://some.different.registry
//npm.pkg.github.com/:_authToken=[TOKEN]
```

Basically what i was looking at it, how we tell npm the respective registry to download from at package level instead of scope level.  Any pointers are appreciated. 
The link i tried is these:

1.  https://docs.npmjs.com/cli/v6/configuring-npm/npmrc
2. https://stackoverflow.com/questions/54543979/npmrc-multiple-registries-for-the-same-scope

### Environment:

- OS: Mac Os Catalina 10.15.7
- Node: v12.10.0
- npm: 6.10.3

**Comment by nlf** [comment]
currently, you can't. there is an RFC to allow for defining registry per-package instead of per-scope, but it has not been ratified or implemented yet. you can follow this issue for updates https://github.com/npm/rfcs/issues/275
